src/test/java/net/mcreator/integration/generator/GTProcedureBlocks.java

```diff
@@ -79,11 +82,38 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 							}
 						}
 
-						if (!match) {
+						if (!match && procedureBlock.getAllRepeatingInputs().stream()

```

Comment:
Is this check really needed? 

This check only checks if certain regular input was not matched by any toolbox_init entry.

If match is here false, it means one of `getAllInputs` does not have template defined, so no need to do extra checks for repeating inputs here.

Reply:
> I don't think what I wrote ignores names of any nodes, not just input_statement ones.

Conversation belongs here for this so keep it here, please.

You did not answer my question why we need this here. It is not that it ignores anything, just how is this needed here.

You iterate regular inputs here and check if every input is defined in toolbox_xml

Reply:
You are right, these are different categories so I shouldn't have been testing them like that. Nothing to fix, though, as you already changed the behavior (👍).

Reply:
Oops, I did not want to commit this change straight away, but it seems it slipped